config/routes/hwi_oauth_routing.yaml

```diff
@@ -1,14 +1,17 @@
 hwi_oauth_redirect:
   resource: "@HWIOAuthBundle/Resources/config/routing/redirect_41.xml"
-  prefix:   /connect
+  prefix:   /{_locale}/connect
 
 hwi_oauth_connect:
   resource: "@HWIOAuthBundle/Resources/config/routing/connect_41.xml"
-  prefix:   /connect
+  prefix:   /{_locale}/connect
 
 hwi_oauth_login:
   resource: "@HWIOAuthBundle/Resources/config/routing/login_41.xml"
-  prefix:   /login
+  prefix:   /{_locale}/login
 
 facebook_login:
+  path: /{_locale}/login/check-facebook
+
+facebook_login_no_locale:
```

Comment:
Why you want to keep login without locale?

Reply:
I used it for testing and forgot to remove it :/